docs-src/zdm-core/modules/migrate/pages/index.adoc

```diff
@@ -0,0 +1,92 @@
+= Introduction to data migration
+:page-tag: migration,zdm,zero-downtime,zdm-proxy,introduction
+//ifdef::env-github,env-browser,env-vscode[:imagesprefix: ../images/]
+//ifndef::env-github,env-browser,env-vscode[:imagesprefix: ]
+
+Enterprises today want to reliably migrate mission-critical client applications and data to cloud environments with zero downtime or near zero downtime during the migration.
+
+A migration is a workflow that encompasses the lifecycle of uploadng and importing your data to the selected keyspaces. 
+At {company}, we've developed a set of thoroughly-tested self-service tools to walk you through well-defined migration options:
+
+* Zero Downtime Migration
+* Cassandra Data Migrations
+* DSBulk Migration
+
```

Comment:
The SSTable sideloader tool can also be included, depending on when this document is released. It is currently under private preview for certain customers. @jeromatron would be able to advise on when it will be available for GA. [SStable sideloader](https://docs.datastax.com/en/astra/astra-db-vector/sideloader/sideloader-overview.html). 
It might be appropriate to replace 'Cassandra Data Migrations' and 'DSBulk Migration' with 'Historic Data Migrations'. Cassandra Data Migrator and DSBulk are tools used to migrate historic data.

Reply:
The sideloader can only be seen in private preview. As soon as the release happens, I can add and publish this information.